Dockerfile

```diff
@@ -375,6 +375,7 @@ ENV DOCKER_BUILDTAGS="${DOCKER_BUILDTAGS}"
 WORKDIR /go/src/github.com/docker/docker
 VOLUME /var/lib/docker
 VOLUME /home/unprivilegeduser/.local/share/docker
+RUN ln -s /usr/local/bin/containerd-shim-runc-v2 /usr/local/bin/containerd-shim-runc-alias-v2
```

Comment:
Can you move this to the test itself?

Reply:
I find it useful to have the test harness around for interactive dev/test use under `make shell`

Reply:
But then you can't run the test without running this.

Reply:
https://github.com/cpuguy83/docker-testing

Maybe the test can just check if it already exists and link it if not?